ThirdPartyNotices.txt

```diff
@@ -3,7 +3,7 @@ THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
 Do Not Translate or Localize
 
 
-The Migrate assets from RM server to Visual Studio Team Services project incorporates third party material from the projects listed below. The original copyright notice and the license under which Microsoft received such third party material are set forth below. Microsoft reserves all other rights not expressly granted, whether by implication, estoppel or otherwise.  
+The Migrate assets from RM server to Visual Studio Team Services project incorporates third party material from the projects listed below. The original copyright notice and the license under which ALM | DevOps Ranger Contributors received such third party material are set forth below. ALM | DevOps Ranger Contributors reserves all other rights not expressly granted, whether by implication, estoppel or otherwise.  
```

Comment:
Can you change "ALM | DevOps Ranger Contributors reserves ..." to "ALM | DevOps Ranger Contributors reserve ..."? Drop the s as Rangers is already multiple.